apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -0,0 +1,199 @@
+
+import locale from './locale';
+import {
+  IS_PARENT_SESSION_KEY,
+  PARENT_EMAIL_SESSION_KEY,
+  PARENT_EMAIL_OPT_IN_SESSION_KEY,
+  DISPLAY_NAME_SESSION_KEY,
+  USER_AGE_SESSION_KEY,
+  USER_STATE_SESSION_KEY,
+  USER_GENDER_SESSION_KEY,
+} from './signUpFlowConstants';
+
+import style from './finishAccount.module.scss';
+
+const FinishStudentAccount: React.FunctionComponent<{
+  ageOptions: {value: string; text: string}[];
+  usStateOptions: {value: string; text: string}[];
+}> = ({ageOptions, usStateOptions}) => {
+  const [isParent, setIsParent] = useState(false);
+  const [parentEmail, setParentEmail] = useState('');
+  const [parentEmailOptInChecked, setParentEmailOptInChecked] = useState(false);
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [state, setState] = useState('');
+  const [gender, setGender] = useState('');
+
+  const onIsParentChange = (): void => {
+    const newIsParentCheckedChoice = !isParent;
+    setIsParent(newIsParentCheckedChoice);
+    sessionStorage.setItem(
+      IS_PARENT_SESSION_KEY,
+      `${newIsParentCheckedChoice}`
+    );
+  };
+
+  const onParentEmailChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    const newParentEmail = e.target.value;
+    setParentEmail(newParentEmail);
+    sessionStorage.setItem(PARENT_EMAIL_SESSION_KEY, newParentEmail);
+  };
+
+  const onParentEmailOptInChange = (): void => {
+    const newParentEmailOptInCheckedChoice = !parentEmailOptInChecked;
+    setParentEmailOptInChecked(newParentEmailOptInCheckedChoice);
+    sessionStorage.setItem(
+      PARENT_EMAIL_OPT_IN_SESSION_KEY,
+      `${newParentEmailOptInCheckedChoice}`
+    );
+  };
+
+  const onNameChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const newName = e.target.value;
+    setName(newName);
+    sessionStorage.setItem(DISPLAY_NAME_SESSION_KEY, newName);
+  };
+
+  const onAgeChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    const newAge = e.target.value;
+    setAge(newAge);
+    sessionStorage.setItem(USER_AGE_SESSION_KEY, newAge);
+  };
+
+  const onStateChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    const newState = e.target.value;
+    setState(newState);
+    sessionStorage.setItem(USER_STATE_SESSION_KEY, newState);
+  };
+
+  const onGenderChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const newGender = e.target.value;
+    setGender(newGender);
+    sessionStorage.setItem(USER_GENDER_SESSION_KEY, newGender);
+  };
+
+  return (
+    <div className={style.finishAccountContainer}>
+      <div className={style.headerTextContainer}>
+        <Heading2>{locale.finish_creating_student_account()}</Heading2>
+        <BodyTwoText>{locale.tailor_experience()}</BodyTwoText>
+      </div>
+      <div className={style.inputContainer}>
+        <div className={style.parentInfoContainer}>
+          <span className={style.parentCheckboxContainer}>
```

Comment:
You can use the `<fieldset>` html element for all of these form label and input pairs. This is better for accessibility 🙌  See more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) in the MDN docs)

Reply:
On it!